coursedata/texts/el.yaml

```diff
@@ -113,8 +113,9 @@ Auth:
     unsaved_changes: 'Έχεις ένα μη αποθηκευμένο πρόγραμμα. Θέλεις να αποχωρήσεις χωρίς να το αποθηκεύσεις;'
     save_success: 'Επιτυχία'
     save_success_detail: 'Το πρόγραμμα αποθηκεύτηκε επιτυχώς'
-    share_success_detail: "Program shared successfully and share link copied to clipboard"
+    share_success_detail: "Program shared successfully"
```

Comment:
Do we need an En translation here? We would just default to En if it is missing right? (also doen not hurt of course...)

Reply:
Right now it is indeed superflous, since a missing text in a non-English defaults to English. However, this entry was already there; all I did was editing it so that the text would be correct, in keeping with the spirit of making the PR as focused as possible. This is the case with a few other languages as well, which have English texts for the `share_success_detail` key.

Reply:
If you want me to remove the untranslated `share_success_detail` entries from the yaml files, let me know and I can do it within this PR.

Reply:
yeah true, they were already there, we can also leave them (someone will work on a better translation UX anyway after Christmas)